include/tvm/ir/function.h

```diff
@@ -63,6 +63,13 @@ enum class CallingConv : int {
    * - Implementation: defined by device runtime(e.g. runtime/cuda)
    */
   kDeviceKernelLaunch = 2,
+  /*!
+   * \brief For functions only called by other functions within the same IR module.
+   *
+   * Indicates that various signature transformations (e.g. those provided by the
+   * MakePackedAPI pass) are not desired.
+   */
+  kIntraModule = 3,
```

Comment:
Would be great to specify that this is a convention that will be lowered (to another low-level convention), or if it is supposed to a final C convention.

Reply:
> Would be great to specify that this is a convention that will be lowered (to another low-level convention), or if it is supposed to a final C convention (with restrictions)

Actually I was planning to leave that unstated, because I wanted the freedom to tweak the details without breaking any TCMScript that uses this.

Do you feel that there's a benefit in being explicit that outweighs that flexibility?